ice/recipes/eval_synthesize.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+# from ice.recipes.synthesize_ft import synthesize_ft_from_df
+# from ice.recipes.synthesize_chain_of_thought import synthesize_chain_of_thought_from_df
+from ice.recipes.synthesize_compositional import synthesize_compositional_from_df
+# from ice.recipes.synthesize import synthesize_from_df
+from ice.recipe import Recipe, recipe
+from ice.recipes.run_over_gs import run_over_gs
+from ice.utils import order_columns
+
+RECIPE_TO_RUN = synthesize_compositional_from_df
```

Comment:
I initially made these parameters to the recipe, rather than just constants, but then got an error when I ran the recipe. My impression is that you can't have dataframes as parameters to a top-level recipe. Not sure if we want to try to change that, for now it's not a big problem